waterlog/src/actions/LeakLitresActions.js

```diff
@@ -0,0 +1,15 @@
+import { FETCH_LEAK_LITRES } from  './types';
+
+export const fetchLeakLitres = (id = 1) => dispatch => {
+  fetch(`https://localhost:44382/api/segmentleaks/litres/${id}`)
```

Comment:
The localhost should be replaced with API_URL from the global.js

Reply:
Replaced it.